server/index.js

```diff
@@ -2,9 +2,10 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var cors = require('cors');
 var path = require('path');
```

Comment:
These could be `const`s as well if you want to keep it consistent

Reply:
Yeah this was here when I got this code, I just keep forgetting to change it!